config/routes.js

```diff
@@ -103,4 +136,6 @@ module.exports = function (app, passport) {
     validator.gameValidation,
     saveGame
   );
+
+  app.get('/api/usergames', users.game);
```

Comment:
Based on the convention prescribed, `usergames` in this endpoint should be in `camelCase`.